### PR TITLE
refactor: fix misc TS errors

### PR DESCRIPTION
**Pull request by olav**

### CODE REVIEW

File: src/component/admin/users/UsersList/DeleteUser/DeleteUser.tsx [hunk 1]
```diff
@@ -33,13 +33,13 @@ const DeleteUser = ({
             open={showDialog}
             title="Really delete user?"
             onClose={closeDialog}
-            onClick={() => removeUser(user)}
+            onClick={removeUser}
```
Comment (FredrikOseberg) [inline src/component/admin/users/UsersList/DeleteUser/DeleteUser.tsx:36]: Nice catch.

File: src/component/common/ResponsiveButton/ResponsiveButton.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { useMediaQuery } from '@material-ui/core';
 import ConditionallyRender from '../ConditionallyRender';
 import PermissionButton from '../PermissionButton/PermissionButton';
 import PermissionIconButton from '../PermissionIconButton/PermissionIconButton';
+import React from 'react';
```
Comment (FredrikOseberg) [inline src/component/common/ResponsiveButton/ResponsiveButton.tsx:5]: Does this cause a tsx error? I thought this was implicit in react 17.

**Review by FredrikOseberg — APPROVED** [review]
Nice. Looks good! Just wondering about the react import. Is this necessary?

File: src/component/common/ResponsiveButton/ResponsiveButton.tsx (hunk 2 above)
Comment (olav) [reply src/component/common/ResponsiveButton/ResponsiveButton.tsx:5]: Oh right!

**Pull request merged**